### ISSUE TITLE
Page templates feature

### ISSUE THREAD

**Issue by brashrebel** [realbigplugins/client-dash-admin-pages#7]
Still need to make the 1.1 milestone which this should go in. But I think it would be sweet to mimic the page template select feature on standard pages and offer it for admin pages. Here's some ideas of _possible_ templates that we could make available:

- Full width (like what we had before #5)
- Card / Column (not sure what to call it but what we went with after #5)
- Widgets (somehow a page which consists entirely of a custom arrangement of admin widgets, picture like a dashboard but focused on a specific topic)
- Settings (I am definitely imagining a day where it is possible to build out a custom settings page, the details will be an entirely separate issue)
- List table

I think we should open each of these and any other templates up as their own issues and make this issue dependent on just the selection option being available and working. Maybe with the first two most basic templates available.

**Comment by joelworsham** [comment]
On that Widgets thing... maybe we could extend the dashboard widget functionality to work on these custom admin pages too? That'd be radical.
